package.json

```diff
@@ -29,11 +29,7 @@
     "lint-fix-css": "yarn lint-css --fix",
     "prettier-run": "node bin/output-fixing-commands.js prettier --check . --cache --cache-strategy content --cache-location .prettiercache",
     "prettier-fix": "prettier --write . --cache --cache-strategy content --cache-location .prettiercache",
-    "flow": "flow --max-warnings 0",
-    "flow:ci": "flow check --max-warnings 0",
-    "flow-stop": "flow stop",
-    "flow-coverage": "npx flow-coverage-report -i 'src/**/*.js' -t html -t text",
-    "flow-generate-libdefs": "npx flow-typed install --libdefDir src/types/libdef",
+    "typecheck": "tsc --noEmit",
```

Comment:
nit: can we maybe name this `ts`? `yarn ts` is shorter and easier to type for me. And we already have the same command in [profiler-server](https://github.com/firefox-devtools/profiler-server/blob/2de28e818a1214d1af9fc3e9ea2e7a76f4ba1697/package.json#L46).

Reply:
Absolutely!